src/core/serialize.cpp

```diff
@@ -564,7 +564,7 @@ void VectorInterface::load_state(ifstream &state)
 
     state.read((char*)&vif_ibit_detected, sizeof(vif_ibit_detected));
     state.read((char*)&vif_interrupt, sizeof(vif_interrupt));
-    state.read((char*)&vif_int_stalled, sizeof(vif_int_stalled));
+    state.read((char*)&vif_stalled, sizeof(vif_stalled));
```

Comment:
vif_stop needs to be implemented here. And since we're doing that, could you increment the "rev" of the savestate version to prevent conflicts with older states?